src/coredump/coredump.c

```diff
@@ -28,6 +28,8 @@
 #include <elfutils/libdwfl.h>
 #endif
 
+#include <eosmetrics/eosmetrics.h>
```

Comment:
After going to the effort of defining an optional dependency on eosmetrics, these things should be #ifdef HAVE_EOSMETRICS otherwise we won't compile without it.